doctrine.rst

```diff
@@ -569,26 +571,18 @@ a controller, this is pretty easy. Add the following method to the
         return new Response('Saved new product with id '.$product->getId());
     }
 
-    // you can also receive the $em as an argument
-    public function editAction(EntityManagerInterface $em)
+    // you can use the registry to fetch other em's, if you have mutiple
+    public function editAction(ManagerRegistry $registry)
```

Comment:
Bah, that's Doctrine's fault. `ManagerRegistry` IS an interface... I think they've only more recently started to use the Interface suffix. 

Btw, I'm hoping the new `debug:container --types` option will help this out - since that valid type-hints